week-4/Homework/mandatory/2-writers.js

```diff
@@ -40,5 +48,61 @@ let writers = [
 ];
 
 /*
-If you want an extra challenge, only `console.log()` the writers that are alive.
+Exercise 1:
+  Loop through the Array, and for each object, `console.log()` out the sentence and
+  insert corresponding values to the place holder that are indicated in courle braces:
+  "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
+
+  console.log ('output using array methods filter and mapping');
+  var data = writers.map (writer => {
+  console.log ('Hi,my name is ' + writer.firstName + ' ' + writer.lastName + ' . I am ' + writer.age + ' years old, and work as a '+writer.occupation);
+  });
+
+  console.log ();
+//let me try for in here
+//struggling with for in here as this is an object array its best to use simple for loop 
+  console.log ('output using for loop');
+  for (var indexes=0; indexes< writers.length;indexes++) {

```

Comment:
```suggestion
  for (let i = 0; i < writers.length; i++) {
```
Ensure you're using the `let` keyword and make sure you've got spaces between your assignment operator `=` and your other variables. 😄